src/model/class.ts

```diff
@@ -151,7 +151,7 @@ export class Class implements Feature {
 
     this.warnings =
         init.warnings === undefined ? [] : Array.from(init.warnings);
-    this.demos = init.demos || jsdocLib.extractDemos(init.jsdoc, document.url);
+    this.demos = init.demos || jsdocLib.extractDemos(init.jsdoc);
```

Comment:
Okay the problem was that init.demos always shows up as an empty array so the || was not useful.  I've opted to combine them now `this.demos = [...init.demos || [], ...jsdocLib.extractDemos(init.jsdoc)]`